### ISSUE TITLE
Project: C olor and l uminance

### ISSUE THREAD

**Issue by estelle** [openwebdocs/project#116]
• [A ccessibility: U nderstanding c olor and l uminance](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_Colors_and_Luminance) was written by a volunteer. She started the article at an accessibility hackathon in 2018 and took a month of concerted effort to write a useful for developers article. It has been completely rewritten over the past 2 years with several links to the author who rewrote it. This needs to be reviewed to see if the content is relevant to our audience or if it should be reverted. 

the original was written by Sharon Austin in 2019. I broke it up into 3 parts:

• [Accessibility: What users can to browse more safely](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Accessibility:_What_users_can_to_to_browse_safely)
• [A ccessibility: U nderstanding c olor and l uminance](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_Colors_and_Luminance)
• [Web accessibility for seizures and physical reactions](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Seizure_disorders)

The second one has deviated from [the original](https://github.com/Myndex/content/blob/9b5cf04786dad342498374e51aacb2ccd1999aac/files/en-us/web/accessibility/understanding_colors_and_luminance/index.html). See the [history](https://github.com/mdn/content/blame/main/files/en-us/web/accessibility/understanding_colors_and_luminance/index.md). It now seems like a portfolio piece for the new author:

> [Page Source](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_Colors_and_Luminance#page_source)
This page is based largely on the white paper on color and contrast by [redacted] which is being developed for the W3 and Accessibility Guidelines Working Group, used herein by permission. [redacted] is an invited expert of the W3 and the AGWG, the author of the WCAG 3 Visual Contrast specifications, and the inventor of the APCA.

**Comment by Elchi3** [comment]
Thanks for filing this OWD project proposal!

We have a project about color docs already. Can accessibility aspects become part of that project? (https://github.com/openwebdocs/project/issues/92)
If not, it would be nice if you could elaborate a bit more what this project is about (and if it is more than updating one page).

**Comment by estelle** [comment]
it's purely about updating one page. The content now has five links to the new main author's research. we need to review it and possibly revert it.

**Comment by Elchi3** [comment]
Should we file this as an MDN issue then as opposed to an OWD project? Trying to get a sense of scope here.

**Comment by Elchi3** [comment]
Closing this as discussed.
